app/main.ipynb

```diff
@@ -248,17 +872,30 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
-   "outputs": [],
-   "source": [
-    "# write your code for task 8 here"
-   ],
+   "source": "data[data[\"hours-per-week\"] == 99][\"age\"].count()",
```

Comment:
you shouldn't write directly `99`, use `max` method: Get the max hours per week some person works. How many people works the same amount of hours per week?